### PR TITLE
fix :add Amandeep404

### LINKED ISSUE #1227: Amandeep Singh
Screenshot 2022-07-13 173946
Screenshot 2022-07-13 174054

### PR DESCRIPTION
**Pull request by Amandeep404**
resolves #1227

## Checklist for profile addition

- [x ] Name the PR `fix: add <your-username>`.
- [x ] You've added file named `your-username.json` in the [profiles](https://github.com/EddieHubCommunity/awesome-github-profiles/tree/main/profiles) directory.
- [x ] You've linked the PR to the correct issue.

---

## Other additions/changes

- [ x] Changes made in this PR are mentioned below
- [x ] Search previous suggestions before making a new one, as yours may be a duplicate.
- [ x] I acknowledge that all my contributions will be made under the project's license.
- [ x] I've read the [contributing guidelines](https://github.com/EddieHubCommunity/awesome-github-profiles/blob/main/CONTRIBUTING.md)

### CODE REVIEW

**Comment by Amandeep404** [comment]
whats the problem in this?

File: profiles/Amandeep.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+    "image": "https://github.com/EddieHubCommunity/awesome-github-profiles/issues/1227#issue-1303348005",
+    "issueId": 1227,
+    "name": "Amandeep Singh",
+    "username": "Amandeep404"
```
Comment (Panquesito7) [inline profiles/Amandeep.json:5]: The indentation seems to be wrong here.
```suggestion
  "image": "https://github.com/EddieHubCommunity/awesome-github-profiles/issues/1227#issue-1303348005",
  "issueId": 1227,
  "name": "Amandeep Singh",
  "username": "Amandeep404"
```


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Inefficient

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Comment by Amandeep404** [comment]
I committed your changes but still the same error...y🥲

**Review by SohamRatnaparkhi — CHANGES_REQUESTED** [review]
Follow a `2 space` indentation for lines `2 to 5` over the 4 space indentation that you have made!

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Inefficient

**Comment by Amandeep404** [comment]
> Follow a `2 space` indentation for lines `2 to 5` over the 4 space indentation that you have made!
> 
> __ [sema-logo](https://semasoftware.com/gh) [ sema-logo ](https://semasoftware.com/gh) [ ](https://semasoftware.com/gh)  **Summary:** 🛠️ This code needs a fix  |  **Tags:** Inefficient

I guess I have given only two space indentation and not 4

**Review by Panquesito7 — APPROVED** [review]
Thank you for your valuable contribution! 🎉
Code looks good. Not sure why the prettier is failing, though. 🤔


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good&nbsp; | &nbsp;**Tags:** Maintainable

**Review by Pradumnasaraf — APPROVED** [review]

**Review by Pradumnasaraf — CHANGES_REQUESTED** [review]
Hey, 
Sorry for approving it at first (The wrong button clicked). Your current screenshot doesn't contain a profile picture also you have to a screenshot of your profile page instead of issue one. So you can update the new one. If you have any issues, let's know

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

**Pull request closed without merging**